doc/man3/OSSL_PARAM_int.pod

```diff
@@ -166,23 +166,22 @@ size B<rsize> is created.
 
 OSSL_PARAM_construct_utf8_string() is a function that constructs a UTF8
 string OSSL_PARAM structure.
-A parameter with name B<key>, storage B<buf>, size B<bsize> and return
-size B<rsize> is created.
+A parameter with name B<key>, storage B<buf> and size B<bsize> is created.
+If B<bsize> is zero, the string length is determined using strlen(3).
```

Comment:
A naked `strlen(3)` should prove interesting.  I just read in the [Pod::Man manual](https://perldoc.perl.org/Pod/Man.html) that it detects and formats these properly, so it should be absolutely fine, really.  It'll be interesting to see how Pod::Html treats it...

Reply:
I'm willing to change it to whatever is required.